bootstrap.example.toml

```diff
@@ -467,6 +467,15 @@
 # Whether to use the precompiled stage0 libtest with compiletest.
 #build.compiletest-use-stage0-libtest = true
 
+# Default value for the `--extra-checks` flag of tidy.
```

Comment:
This PR is becoming quite large, and this looks quite self-contained (and useful on its own). Could you please land it separately?